server/README.md

```diff
@@ -4,4 +4,16 @@
 _Note: Requires that [poetry](https://python-poetry.org/) be installed._
 
 - `poetry install` to download all required dependencies and set up a virtual environment
-- `poetry run task run` to start the Django server
+- `poetry run flask run` to start the Flask server
```

Comment:
`poetry run task run` runs the `run` task in the pypoetry.toml file (under `[tool.taskipy.tasks]`) so you can just add more tasks there for db migration (also, the `run` task is already configured for `flask run`)

_phew, lots of runs and tasks in that sentence_ 😝

Reply:
I added a couple of new _tasks_:
- `add-migration` => `flask db migrate` (looks like command line args get passed from the `taskipy` docs)
- `apply-migrations` => `flask db upgrade` (debated making this a pre_task of the "run task", but I do like that a schema migration needs to be an explicit "task"—sorry to use this overused word—that is run)

Reply:
Not going to add `revert-migrations` or anything like that because 🤞 hopefully we don't ever need to use them 😄